### ISSUE TITLE
ModelNotFoundException should be transformed into NotFoundHttpException

### ISSUE THREAD

**Issue by Sly-Ripper** [GrahamCampbell/Laravel-Exceptions#49]
Default behavior in 5.2 now

``` php
namespace GrahamCampbell\Exceptions\Transformers;

use Exception;
use Illuminate\Database\Eloquent\ModelNotFoundException;
use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;

/**
 * This is the auth transformer class.
 *
 * @author Graham Campbell <graham@alt-three.com>
 */
class ModelNotFoundTransformer implements TransformerInterface
{
    /**
     * Transform the provided exception.
     *
     * @param \Exception $exception
     *
     * @return \Exception
     */
    public function transform(Exception $exception)
    {
        if ($exception instanceof ModelNotFoundException) {
            $message = $exception->getMessage();
            $exception = new NotFoundHttpException($message ?: null);
        }

        return $exception;
    }
}
```

**Comment by GrahamCampbell** [comment]
Feel free to add that behaviour if you want it. I don't like it though as a default, because as far as i'm concerned, such an error might still be an internal server error. I prefer to catch and rethrow in the code itself.

**Comment by GrahamCampbell** [comment]
Maybe we should ship with a transformer enabled by default though.

**Comment by GrahamCampbell** [comment]
> Default behavior in 5.2 now

This was true in 5.1 too.

**Comment by simplenotezy** [comment]
Is there some workaround I can implement now?

**Comment by GrahamCampbell** [comment]
Give me a few mins.

**Comment by GrahamCampbell** [comment]
This will be fixed in the 8.3 release. Just update your exceptions.php config file to match the new one I just updated.

**Comment by eyyubzade** [comment]
why?
